docs/ingestion/data-management.md

```diff
@@ -21,173 +21,9 @@ title: "Data management"
   ~ specific language governing permissions and limitations
   ~ under the License.
   -->
+Within the context of this topic Data management refers to Apache Druid's data maintenance capabilities for existing datasources. There are several options to help you keep your data relevant and to help your Druid cluster remain performant. For example updating, reingesting, adding lookups, reindexing, or deleting data.
```

Comment:
nit: this line mentions "existing datasources" and the first topic in this page is `Adding new data`. I don't have a good idea yet on what the correct split should be yet.

Reply:
@suneet-s , that means adding new data to existing data sources. I will change that heading.